myapp/src/components/Button.js

```diff
@@ -10,11 +10,9 @@ const useStyles = makeStyles(theme => ({
     }
 }))
 
-export default function Button(props) {
+const Button = ({ text, size, color, variant, onClick, ...other }) => {
```

Comment:
Remove ...other use only the props you need

Reply:
I tried to do this before but then the button wont have the proper styles.